### PR TITLE
Add namespace allowlist to trivy

### PR DESCRIPTION
**Pull request by rbren**
## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What's the goal of this PR?
Add an allowlist in addition to the blocklist. Also change blacklist to blocklist

### CODE REVIEW

File: plugins/trivy/pkg/image/getimages.go [hunk 1]
```diff
@@ -17,6 +17,32 @@ import (
...
+var namespaceAllowlist []string
+
+func init() {
+	namespaceBlocklist = strings.Split(os.Getenv("NAMESPACE_BLACKLIST"), ",")
+	namespaceBlocklist = append(strings.Split(os.Getenv("NAMESPACE_BLOCKLIST"), ","), namespaceBlocklist...)
+	namespaceAllowlist = strings.Split(os.Getenv("NAMESPACE_ALLOWLIST"), ",")
+}
+
+func namespaceIsBlocked(ns string) bool {
+	for _, namespace := range namespaceBlocklist {
```
Comment (jdesouza) [inline plugins/trivy/pkg/image/getimages.go:30]: No need to lower case ns?

File: plugins/trivy/pkg/image/getimages.go [hunk 2]
```diff
@@ -17,6 +17,32 @@ import (
...
+		if ns == strings.TrimSpace(strings.ToLower(namespace)) {
+			return true
+		}
+	}
+	if len(namespaceAllowlist) == 0 {
+		return false
+	}
+	for _, namespace := range namespaceBlocklist {
```
Comment (jdesouza) [inline plugins/trivy/pkg/image/getimages.go:38]: Should not this be the allowed list?

Comment (rbren) [inline plugins/trivy/pkg/image/getimages.go:38]:
```suggestion
	for _, namespace := range namespaceAllowlist {
```

File: plugins/trivy/pkg/image/getimages.go (hunk 1 above)
Comment (rbren) [reply plugins/trivy/pkg/image/getimages.go:30]: Nah that's coming straight from the k8s API, so it'll always be lower

**Review by jdesouza — APPROVED** [review]

**Pull request merged**